.eslintrc.js

```diff
@@ -0,0 +1,66 @@
+module.exports = {
```

Comment:
you should have ppl install this plugin, it should let them know in their ide if there's an eslint error (might need to be configured a lil tho)
<img width="452" alt="image" src="https://github.com/Northeastern-Electric-Racing/Argos/assets/29521172/5a8451ab-f07c-40b9-a6b1-117ae0df7b20">